### ISSUE TITLE
Can't open folder from breadcrumbs in secondary view

### ISSUE THREAD

**Issue by ElijahLynn** [microsoft/vscode#224898]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
```
code-insiders --version
1.93.0-insider
39d98602020ddefe53ed217a8c50b9040673a737
arm64
```

- OS Version: 
```
14.5 (23F79)
```

Steps to Reproduce:

1. Click on a foldername in the breadcrumb, or double click it, or right click it
2. No way for me to open the folder in the secondary sidebar in my folder/file explorer

**Comment by ElijahLynn** [comment]
That is great! I wonder if we can combine the upvotes already on:
* https://github.com/microsoft/vscode/issues/67711

**Comment by montapro** [comment]
Yes, this is the same requirement. Please combine the upvotes. 🙏

**Comment by david-bakin-sl** [comment]
Say, are you supposed to upvote the original issue or the first bot comment where it asks for votes?

**Comment by ElijahLynn** [comment]
> Say, are you supposed to upvote the original issue or the first bot comment where it asks for votes?

lol, I'm _hoping_ it is the original issue! 🤞

**Comment by ElijahLynn** [comment]
I think I found a workaround, `cmd + shift + >` opens a dropdown, use arrows to move up, down, left, right, then if you have a directory highlighted (or file), press `cmd + return` and it will open the folder or file in the file explorer.
